README.md

```diff
@@ -11,4 +13,7 @@
 1. run `npm run dev` to start client
 
 ## Documentation
-- [Server architecture](server/ARCHITECTURE.md)
+
+- [Client README.md](client/README.md)
+- [Server README.md](server/README.md)
+- [Server ARCHITECTURE.md](server/ARCHITECTURE.md)
```

Comment:
nit: is the `.md` important to include here?

Reply:
agreed, and changed.